features/karaf/src/main/feature/feature.xml

```diff
@@ -12,8 +12,8 @@
 <features name="${project.artifactId}-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.0.0">
 
     <feature name="openhab-runtime-base" description="openHAB Runtime Base" version="${project.version}">
-        <feature>shk-esh-base</feature>
-        <feature>shk-esh-io-console-karaf</feature>
+        <feature>esh-base</feature>
```

Comment:
As esh-base has been stripped off, you will perhaps add some other features, e.g.
```text
    <feature>esh-base</feature>
    <feature>esh-io-console-karaf</feature>
    <feature>esh-io-rest-sitemap</feature>
    <feature>esh-model-item-runtime</feature>
    <feature>esh-model-persistence-runtime</feature>
    <feature>esh-model-rule-runtime</feature>
    <feature>esh-model-script-runtime</feature>
    <feature>esh-model-sitemap-runtime</feature>
    <feature>esh-model-thing-runtime</feature>
    <feature>esh-storage-mapdb</feature>
    <feature>esh-ui-iconset-classic</feature>
```